docs/content/customization/generic/dns.md

```diff
@@ -15,6 +15,35 @@ If the `dns.coreDNS` property is not specified, then the customization will be s
 
 ### Example
 
+The CoreDNS version can be updated automatically. To do this, set `coreDNS` to an empty object:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: <NAME>
+spec:
+  topology:
+    variables:
+      - name: clusterConfig
+        value:
+          dns:
+            coreDNS: {}
```

Comment:
we should provide both image and tag in the example to illustrate how it will transform to `KubeadmControlPlaneTemplate`

Reply:
@supershal I think that example is already defined further down on this page?

Reply:
I see now.